lib/plivo/resources/multipartycalls.rb

```diff
@@ -189,6 +197,26 @@ def resume_recording
         perform_action_apiresponse('Record/Resume', 'POST')
       end
 
+      def start_participant_recording(member_id, file_format='mp3', status_callback_url=nil, status_callback_method='POST')
+        valid_param?(:member_id, member_id, [String, Integer], true)
+        MultiPartyCallParticipant.new(@_client, resource_id: mpc_id[1], member_id: member_id).start_participant_recording(file_format, status_callback_url, status_callback_method)
```

Comment:
Why is this mpc_id[1]? It should have been @id, refer the same for get_participant, kick_participant functions

Reply:
@Koushik-Ayila  yep, sorry for that.  Updated them now.